specification/programmableconnectivity/data-plane/Azure.ProgrammableConnectivity/preview/2024-02-09-preview/openapi.json

```diff
@@ -867,10 +858,15 @@
         "hashedPhoneNumber": {
           "type": "string",
           "description": "Hashed phone number. SHA-256 (in hexadecimal representation) of the mobile phone number in **E.164 format (starting with country code)**. Optionally prefixed with '+'."
+        },
+        "redirectUri": {
+          "type": "string",
```

Comment:
I think (bit I'm not sure) that the type for this should be uri and not string - please check

Reply:
`type:string, format:uri` is how uris are represented in swagger

Reply:
In the `.tsp` file, I have

```
  @doc("Redirect URI to backend application.")
  redirectUri: url;
```

This file is completely auto-generated. Can you confirm whether you're saying that I should be manually changing this from a `string` here?

Reply:
Now
```
        "redirectUri": {
          "type": "string",
          "format": "uri",
          "description": "Redirect URI to backend application."
        }
```
        As desired